lib/ansible/modules/cloud/amazon/cloudformation_stack_set.py

```diff
@@ -0,0 +1,664 @@
+    status: FAILED
+stack_instances:
+  description: CloudFormation stack instances that are members of this stack set. This will also include their region and account ID.
+  returned: state == present
+  type: list
+  sample:
+    - account: '1234567890'
+      region: us-east-1
+      stack_set_id: TestStackPrime:19f3f684-aae9-4e67-ba36-e09f92cf5929
+      status: OUTDATED
+      status_reason: >
+        Account 1234567890 should have 'AWSCloudFormationStackSetAdministrationRole' role with trust relationship to CloudFormation service.
+    - account: '1234567890'
+      region: us-east-2
+      stack_set_id: TestStackPrime:19f3f684-aae9-4e67-ba36-e09f92cf5929
+      status: OUTDATED
+      status_reason: Cancelled since failure tolerance has exceeded
+stack_set:
+  type: dict
+  description: Facts about the currently deployed stack set, its parameters, and its tags
+  returned: state == present
+  sample:
+    administration_role_arn: arn:aws:iam::1234567890:role/AWSCloudFormationStackSetAdministrationRole
+    capabilities: []
+    description: test stack PRIME
+    execution_role_name: AWSCloudFormationStackSetExecutionRole
+    parameters: []
+    stack_set_arn: arn:aws:cloudformation:us-east-1:1234567890:stackset/TestStackPrime:19f3f684-aae9-467-ba36-e09f92cf5929
+    stack_set_id: TestStackPrime:19f3f684-aae9-4e67-ba36-e09f92cf5929
+    stack_set_name: TestStackPrime
+    status: ACTIVE
+    tags:
+      Some: Thing
+      an: other
+    template_body: |
+      AWSTemplateFormatVersion: "2010-09-09"
+      Parameters: {}
+      Resources:
+        Bukkit:
+          Type: "AWS::S3::Bucket"
+          Properties: {}
+        other:
+          Type: "AWS::SNS::Topic"
+          Properties: {}
+
+'''  # NOQA
+
+import time
+import datetime
+import uuid
+import itertools
+
+try:
+    import boto3
+    import botocore.exceptions
+    from botocore.exceptions import ClientError, BotoCoreError
+except ImportError:
+    # handled by AnsibleAWSModule
+    pass
+
+import ansible.module_utils.ec2
+# import a class, otherwise we'll use a fully qualified path
+from ansible.module_utils.ec2 import AWSRetry, boto3_tag_list_to_ansible_dict, ansible_dict_to_boto3_tag_list, camel_dict_to_snake_dict
+from ansible.module_utils.aws.core import AnsibleAWSModule, is_boto3_error_code
+from ansible.module_utils._text import to_native
+
+
+def create_stack_set(module, stack_params, cfn):
+    try:
+        cfn.create_stack_set(aws_retry=True, **stack_params)
+        return await_stack_set_exists(cfn, stack_params['StackSetName'])
+    except (ClientError, BotoCoreError) as err:
+        module.fail_json_aws(err, msg="Failed to create stack set {0}.".format(stack_params.get('StackSetName')))
+
+
+def update_stack_set(module, stack_params, cfn):
+    # if the state is present and the stack already exists, we try to update it.
+    # AWS will tell us if the stack template and parameters are the same and
```

Comment:
It runs, but it makes it so we can't tell if changed should be True or not, it seems. I wish it raised an exception instead.